### PR TITLE
Resolve Issue #142, #89, and IllegalStateException warning.

### PR DESCRIPTION
**Pull request by JediBurrell**
These changes fix Issue #142 by preventing the comment EditText from catching focus on creation.

### CODE REVIEW

File: app/src/main/res/layout/dribbble_enter_comment.xml [hunk 1]
```diff
@@ -51,6 +51,7 @@
             android:layout_height="wrap_content"
             android:layout_weight="1"
             android:inputType="textCapSentences"
+            android:focusable="false"
```
Comment (ataulm) [inline app/src/main/res/layout/dribbble_enter_comment.xml]: This would break keyboard focus (keyboard access won't trigger `onTouch`)

Comment (JediBurrell) [inline app/src/main/res/layout/dribbble_enter_comment.xml]: @ataulm Correct. That's why in DribbbleShot.java (Line 713-721), it sets it back to true.

Worked fine when I tested it.

Comment (ataulm) [inline app/src/main/res/layout/dribbble_enter_comment.xml]: It sets it back to true in onTouch, which won't be triggered for keyboard users

Comment (JediBurrell) [inline app/src/main/res/layout/dribbble_enter_comment.xml]: @ataulm What do you mean?

To gain focus, you have to touch the `EditText` field.
The initial focus ( On screen keyboard, more realistically ) messes with the layout.

Comment (ataulm) [inline app/src/main/res/layout/dribbble_enter_comment.xml]: Sorry, I wasn't clear 🙂 

You can focus both in touch or non-touch mode, where the former is possible if a View `focusableInTouchMode` property is set to true (as it is for EditText).

You could try this fix by modifying the `focusableInTouchMode` property for example and it should not affect hardware keyboard access.

If the error is that the soft keyboard is messing with the layout when you first open the activity, perhaps setting `android:windowSoftInputMode="stateHidden"` for this Activity in the manifest might be enough?

Comment (JediBurrell) [inline app/src/main/res/layout/dribbble_enter_comment.xml]: @ataulm That makes sense, I've just pushed a new commit that instead utilizes `android:windowSoftInputMode="stateHidden"` and it works just as well, if not ( definitely ) better. :)

**Comment by nickbutcher** [comment]
Thanks for the contribution but I went another way and resolved this in f1b8bd8925e98dca4c5a5b83303dc2ba3d83e3d2

**Pull request closed without merging**